plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/Core.melk

```diff
@@ -30,24 +30,23 @@ import org.eclipse.elk.core.util.RandomLayoutProvider
 /**
  * Core definitions of the Eclipse Layout Kernel.
  */
-bundle {
-	label "ELK"
-	metadataClass core.options.CoreOptions 
+bundle{
+    label "ELK"  
+    metadataClass core.options.CoreOptions 
 	idPrefix org.eclipse.elk 
 }
 
-category layered {
-	label "Layered"
-	description
+category layered {	
+    label "Layered"	description
```

Comment:
This looks odd, there should be newline before description, right?

Reply:
Yep.